pkg/generate/rand.go

```diff
@@ -70,3 +70,13 @@ func randomBytes(src rand.Source, bytes string, n int) []byte {
 
 	return b
 }
+
+// RandomInt returns a generated int in the range >= min, <= max
+func RandomInt() int {
+	var (
+		min = 1
+		max = 100
+	)
+	rand.Seed(time.Now().UnixNano())
```

Comment:
This sets the default seed for the package, while it won't be significant for performance, calling it on every function invocation is a bit excessive. An option would be to have a wrapper that returns the actual function we need, which in turn could accept a `Rand` and if not given would init its own. For example:

``` go
type RandPercentageFunc func() int

func RandPercentage(r *rand.Rand) RandPercentageFunc {
    seed := rand.New(time.Now().UnixNano())

    return func() int {
        return seed.Intn(99) + 1
    }
}
```